### PR TITLE
[FIX] 14.0 mass_edit several attribute in a group

### PR DESCRIPTION
**Pull request by PierrickBrun**

### CODE REVIEW

**Review by hparfr — APPROVED** [review]

File: attribute_set_mass_edit/models/attribute_group.py [hunk 1]
```diff
@@ -28,3 +28,12 @@ def write(self, vals):
                     # mass_editing.disable_mass_operation()
                     # mass_editing.enable_mass_operation()
         return res
+
+    def _prepare_create_mass_editing(self):
+        self.ensure_one()
+        return {
+            "mass_edit_attribute_group_id": self.id,
+            "model_id": self.model_id.id,
+            "name": self.name,
```
Comment (sebastienbeau) [inline attribute_set_mass_edit/models/attribute_group.py]: We should put a better name for the mass edition
```suggestion
            "name": _("Edit %s Fields") % self.name,
```

**Comment by sebastienbeau** [comment]
@PierrickBrun can you check my change ?
image

**Review by sebastienbeau — APPROVED** [review]
LGTM (code review and test)

**Comment by PierrickBrun** [comment]
> @PierrickBrun can you check my change ? image

Thanks !

**Pull request merged**